sfs/mono/__init__.py

```diff
@@ -3,14 +3,23 @@
 .. autosummary::
     :toctree:
 
-    drivingfunction
+    esaedge
+    nfchoa
+    sdm
+    wfs
+
     source
     soundfigure
```

Comment:
I'm not sure if it should happen within this PR, but what about moving this to the `wfs` module?

IOW, move `sfs.mono.soundfigure.wfs_3d_pw()` to `sfs.mono.wfs.soundfigure_something()`?

Reply:
moved to sfs.mono.wfs.soundfigure_2d()
I am not sure, it might even be a _3d scenario, can you @spors validate naming.

Note: this code is not tested at all in the moment, so I'm going to open a new issue for this. But for now the handling would be more consistent.